--- apachetop-0.12.6/src/apachetop.h~	2011-11-08 17:17:58.000000000 +0100
+++ apachetop-0.12.6/src/apachetop.h	2011-11-08 17:18:22.621108652 +0100
@@ -253,7 +253,7 @@
 
 /* this can be overridden from config.h via ./configure --with-logfile .. */
 #ifndef DEFAULT_LOGFILE
-# define DEFAULT_LOGFILE "/var/log/apache2/access.log"
+# define DEFAULT_LOGFILE "/var/log/httpd/access.log"
 #endif
 #define DEFAULT_CIRCLE_SIZE 30
 #define DEFAULT_CIRCLE_MODE TIMED_CIRCLE
